web/src/components/common/DeadlineTag.tsx

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Not related to this file specifically but to the folder. Why is it named as "common"? Are items like DeadlineTag, ListingCard, ListingPrice going to be re-used in more than one places?

Reply:
@xlai20 Yes they are going to be reused if they are in `/common/`! Can refer to https://github.com/googleinterns/gpay-group-buy/blob/master/web/README.md#project-structure for the structure we have decided on